src/LowMachEquationSystem.C

```diff
@@ -1177,6 +1179,12 @@ MomentumEquationSystem::register_nodal_fields(
     stk::mesh::put_field_on_mesh(*actuatorSourceLHS, *part, nDim, nullptr);
     stk::mesh::put_field_on_mesh(*g, *part, nullptr);
   }
+
+  GenericFieldType& node_mask =
+    realm_.meta_data().declare_field<GenericFieldType>(
+      stk::topology::NODE_RANK, "abl_wall_no_slip_wall_func_node_mask");
+  double one = 1;
+  stk::mesh::put_field_on_mesh(node_mask, *part, 1, &one);
```

Comment:
Minor nitpick. This can just be `ScalarFieldType` as it is implemented on nodes and not on edges/elements.

Reply:
Done.